artemis-server/src/main/java/org/apache/activemq/artemis/core/server/cluster/impl/ClusterConnectionImpl.java

```diff
@@ -630,6 +630,11 @@ public long getCallTimeout() {
       return callTimeout;
    }
 
+   @Override
+   public Bridge[] getBridges() {
+      return records.values().stream().map(MessageFlowRecord::getBridge).toArray(Bridge[]::new);
```

Comment:
The 'records' field has 'recordsGuard' to protect its use in other cases. Is that needed here?

Reply:
It makes sense, I'll fix it. I'll create another PR to fix it.